backstop_data/engine_scripts/casper/onBefore.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function (casper, scenario, vp) {
+  require('./loadCookies')(casper, scenario);
+  casper.userAgent('Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36');
```

Comment:
isn't this line already provided by the loadCookies file?

https://github.com/compucorp/backstopjs-config/pull/7/files#diff-f552e1645b116455586a8eca5019c0f3R14

Reply:
again, backstopjs stuff (anyway the whole casper/ dir will be gone in my next ticket)